src/components/ui/DropdownMenu.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <label>Sort by:</label>
+  <!-- extrenal library for Select styling?? Icon up and down should be added -->
+  <select name="choice">
+    <option value="most upvotes">Most upvotes</option>
+    <option value="least upvotes">Least upvotes</option>
+    <option value="most comments">Most comments</option>
+    <option value="least comments">Least comments</option>
```

Comment:
I think you should make one BaseSelect component and pass the values as props :)

Reply:
I created BaseSelect and BaseOption to have the possibility to use different options in different components with an array of values and v-for directives. Is it the right way or it is possible to make it in one component? 

Another question is that the value and the text inside the option tag are the same - is it ok that I put one value for 2 of them (so I created an array instead of an object with 2 key-value pairs)? (eg. I am not sure if the value can have capital letters).